scripts/bootstrap-aio.yml

```diff
@@ -272,3 +272,98 @@
     rpco_deploy_ceph: "{{ lookup('env', 'DEPLOY_CEPH') }}"
     rpco_deploy_elk: "{{ lookup('env', 'DEPLOY_ELK') }}"
     rpco_deploy_swift: "{{ lookup('env', 'DEPLOY_SWIFT') }}"
+
+- name: Execute ironic pre-config steps
+  hosts: localhost
+  tasks:
+    - name: Create ironic network configurations
+      blockinfile:
+        dest: /etc/network/interfaces.d/osa_ironic_interfaces.cfg
+        create: yes
+        block: |
+          auto none
+          auto br-ironic-ipmi
+          iface br-ironic-ipmi inet static
+              bridge_ports none
+              bridge_stp off
+              bridge_waitport 10
+              bridge_fd 0
+              offload-sg off
+              address 172.29.228.100
+              netmask 255.255.252.0
+
+          auto none
+          iface none inet manual
+          auto br-tftp
+          iface br-tftp inet static
+              bridge_stp off
+              bridge_waitport 10
+              bridge_fd 0
+              offload-sg off
+              bridge_ports none
+              address 172.29.232.100
+              netmask 255.255.252.0
+      when: "{{ lookup('env', 'DEPLOY_IRONIC') == 'yes' }}"
```

Comment:
Ansible gets upset if you put jinja2 delimiters in `when:` clauses. You might want to put something like this up above where you see `DEPLOY_SWIFT`:

```yaml
rpco_deploy_ironic: "{{ lookup('env', 'DEPLOY_IRONIC') }}"
```

And then use `when: rpco_deploy_ironic == 'yes'` or `when: rpco_deploy_ironic | bool` in your tasks.